crlite_query/query_cli.py

```diff
@@ -176,18 +176,27 @@ def main():
     to_test = list()
 
     for file in args.files:
-        to_test.append((file.name, query.pem(file)))
+        to_test.append((file.name, query.gen_from_pem(file)))
 
     for host_str in args.hosts:
         parts = host_str.split(":")
         hostname = parts[0]
         port = 443
         if len(parts) > 1:
             port = int(parts[1])
-        to_test.append((f"{hostname}:{port}", query.host(hostname, port)))
+        to_test.append((f"{hostname}:{port}", query.gen_from_host(hostname, port)))
 
-    for (name, iterator) in to_test:
-        query.print_query(name=name, iterator=iterator)
+    failures = list()
+
+    for (name, generator) in to_test:
+        for result in query.query(name=name, generator=generator):
+            result.print_query_result()
+
+    if failures:
```

Comment:
Were we supposed to append something to `failures`?

Reply:
Ooo, thank you, something in the rebase killed those lines.

Reply:
https://github.com/mozilla/moz_crlite_query/commit/4b701c2 had it

Reply:
Huh, it's there, see https://github.com/mozilla/moz_crlite_query/pull/8/files#diff-48f6a13a01f0caf2caba6154cb3759c4R200-R201

Reply:
Ah, my bad!